app/main.py

```diff
@@ -1,8 +1,46 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+
+    def __init__(self,
+                 distance_from_city_center: float,
+                 clean_power: int,
+                 average_rating: float,
+                 count_of_ratings: int) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def wash_single_car(self, car: Car) -> None:
+        if car.clean_mark < self.clean_power:
+            car.clean_mark = self.clean_power
+
+    def calculate_washing_price(self, car: Car) -> float:
+        if self.clean_power > car.clean_mark:
+            return round((car.comfort_class
+                          * (self.clean_power - car.clean_mark)
+                          * self.average_rating
+                          / self.distance_from_city_center), 1)
+
+    def serve_cars(self, cars: list[Car]) -> float:
```

Comment:
TypeError: 'type' object is not subscriptable
please, instead try cars: list

Reply:
@AnnaBoichenko do not clearly understand your point.
I used this [example](https://docs.python.org/3/library/typing.html#:~:text=type%20inside%20TypeGuard.-,for%20example%3A,-def%20is_str_list(val) to format my code. 
Could you clarify your POV, please?

Reply:
@AnnaBoichenko Do not understand your comment.
Could you clarify your POV, please?
I used [this example](https://docs.python.org/3/library/typing.html#:~:text=type%20inside%20TypeGuard.-,for%20example%3A,-def%20is_str_list(val) to format my code